I wish_If only/Bohdanov_hw.md

```diff
@@ -0,0 +1,41 @@
+# Exercise 1. Please translate the sentences into English
+
+>Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+
+If only you had came up earlier to (evaluate / estimate) all the lameness of architecture that was proposed by first year student.
```

Comment:
If we talk about the past in this case we are supposed to use Past Perfect - `I wish you had come` in Past Perfect we use the third form of the verb which is the same as the initial form for the verb come.
`the architecture` - as we precise what kind of architecture we are talking about